packages/wallet/src/redux/channelState/funding/reducer.ts

```diff
@@ -41,8 +31,6 @@ export const fundingReducer = (
     // Setup funding process
     case states.WAIT_FOR_FUNDING_REQUEST:
```

Comment:
Consider that the `indirectFundingReducer` will need to react to an action to get into the `WAIT_FOR_APPROVAL` state. What action could that be? I think the only sensible action is `FUNDING_REQUESTED`.

Therefore, you might also want to remove this state, leaving the channel status in the `WAIT_FOR_FUNDING_AND_POST_FUND_SETUP` stage and leaving it up to the funding-related code to deal with this.

Reply:
I agree with this thinking. https://github.com/magmo/apps/pull/301